apps/explorer/src/app/components/txs/details/chain-events/tx-builtin-deposit.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
comment (non-blocking): importing react is not necessary when not used anymore

Reply:
Fixed in all the new files, will progressively do this throughout the codebase